firefox_puppeteer/ui/toolbars.py

```diff
@@ -446,3 +446,20 @@ def verifier(self):
         :returns: Reference to the identity-popup content verifier.
         """
         return self.marionette.find_element(By.ID, 'identity-popup-content-verifier')
+
+    def close(self, force=False):
+        """Closes the identity popup by hitting the escape key.
+
+        :param force: Optional, flag that, if true, will force close the popup.
+        """
+        if not self.is_open:
+            return
+
+        if force:
+            self.marionette.execute_script("""
+                arguments[0].hidePopup();
+                """, script_args=[self.popup])
```

Comment:
The formatting of the code has not been updated as pointed out in my review on PR #105.